docsite/rst/playbooks_blocks.rst

```diff
@@ -56,8 +55,8 @@ Blocks also introduce the ability to handle errors in a way similar to exception
        - debug: msg="this always executes"
 
 
-The tasks in the ``block`` would execute normally, if there is any error the ``rescue`` section would get executed
-with whatever you need to do to recover from the previous error. The ``always`` section runs no matter what previous
+The tasks in the ``block`` section will execute normally.  If there is an error, the ``rescue`` section will be executed;
+here place whatever you need to do to recover from the previous error. The ``always`` section runs no matter what previous
```

Comment:
I'm not a fan of this line, could you tweak it to be a bit clearer? Starting a new sentence rather than using a semicolon would probably be easier to read.